sidebar/window.c

```diff
@@ -686,30 +693,34 @@ int sb_recalc(struct MuttWindow *win)
 
     const char *path = mailbox_path(m);
 
+		const char const c_folder = cs_subset_path(sub, "folder");
     // Try to abbreviate the full path
-    const char *abbr = abbrev_folder(path, C_Folder, m->type);
+    const char *abbr = abbrev_folder(path, c_folder, m->type);
     if (!abbr)
       abbr = abbrev_url(path, m->type);
     const char *short_path = abbr ? abbr : path;
 
     /* Compute the depth */
     const char *last_part = abbr;
-    entry->depth = calc_path_depth(abbr, C_SidebarDelimChars, &last_part);
+    entry->depth = calc_path_depth(abbr, c_sidebar_delim_chars, &last_part);
 
     const bool short_path_is_abbr = (short_path == abbr);
-    if (C_SidebarShortPath)
+		const bool c_sidebar_short_path = cs_subset_bool(sub, "sidebar_short_path");
+    if (c_sidebar_short_path)
     {
       short_path = last_part;
     }
 
     // Don't indent if we were unable to create an abbreviation.
     // Otherwise, the full path will be indent, and it looks unusual.
-    if (C_SidebarFolderIndent && short_path_is_abbr)
+		const bool c_sidebar_folder_indent = cs_subset_bool(sub, "sidebar_folder_indent");
+    if (c_sidebar_folder_indent && short_path_is_abbr)
     {
-      if (C_SidebarComponentDepth > 0)
-        entry->depth -= C_SidebarComponentDepth;
+      const short c_sidebar_component_depth = cs_subset_number(sub, "sidebar_component_depth");
+			if (c_sidebar_component_depth > 0)
+				entry->depth -= c_sidebar_component_depth;
```

Comment:
Spaces please, rather than tabs.
In vim:
``vim
set tabstop=2 shiftwidth=2 expandtab
```